links.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft/hyper-schema#",
```

Comment:
Because this is for "master" so it's not a numbered draft (at least that's how @awwright set it up and I was trying to not change it- take a look at the "id" of the hyper-schema).